rate_counter/rate/tasks.py

```diff
@@ -96,3 +104,37 @@ def get_url(inst='1', group='', semestr='1', semestr_part='1'):
                                                                                                 semestr,
                                                                                                 semestr_part,
                                                                                                 )
+
+
+def rate_graphs(id):
```

Comment:
rate_graphs - it is hard to understand what this function is doing by its name. 
It woud be better smth like 'prepare_column_data'